### PR TITLE
adding missing documentation and readinessProbe

### LINKED ISSUE #7569: [stable/minio]  service.externalIPs is not defined in values.yaml and other lint issues
Lint issues: 

1.  Parameter is used, but not defined: service.externalIPs is not defined, but used in [ibm-minio-objectstore/templates/service.yaml.  This is needed even as documentation purpose in values.yaml because if user  wants to use it, he is clueless how to specify it. 

2. Parameter is used, but not defined: persistence.existingClaim is not defined, but used in [ibm-minio-objectstore/templates/deployment.yaml ibm-minio-objectstore/templates/pvc.yaml

3. Missing readinessProbe: readinessProbe not found for container 0 in ibm-minio-objectstore/templates/deployment.yaml

### PR DESCRIPTION
**Pull request by cpanato**
**Which issue this PR fixes** : fixes https://github.com/helm/charts/issues/7569
**Special notes for your reviewer**:

### CODE REVIEW

File: stable/minio/templates/deployment.yaml [hunk 1]
```diff
@@ -93,6 +93,11 @@ spec:
                   key: gcs_key.json
             {{- end }}
           livenessProbe:
+            tcpSocket:
```
Comment (sanverm2) [inline stable/minio/templates/deployment.yaml:96]: You may also need to add both livenessProbe  and readinessProbe  in templates/statefulset.yaml.

Comment (cpanato) [reply stable/minio/templates/deployment.yaml:96]: will add, thanks!

Comment (cpanato) [reply stable/minio/templates/deployment.yaml:96]: https://github.com/helm/charts/pull/7598

**Pull request merged**